Deliverable2/Program.cs

```diff
@@ -4,79 +4,76 @@
     public static void Main(string[] args)
     {
         // Set Variables: Guest Limit == 6, Buffet Cost == $9.99, water == Free, Softdrink == $2.00//
-        int GuestCount = 0;
-        double SoftDrinkTotal = 0;
-        double WaterTotal = 0;
-        bool ReEnterGroupSize = false;
-        bool DoNotRunReceipt = false;
-        string Padding = "    ";
+        double softDrinkTotal = 0;
+        double waterTotal = 0;
+        var reEnterGroupSize = false;
+        const string padding = "    ";
 
         //Welcome Guests//
         Console.ForegroundColor = ConsoleColor.Blue;
         Console.WriteLine("Welcome to Greasy Jean's All You Can Eat Food Trough!");
-        Console.WriteLine("This all you can eat experiance only costs $9.99 per guest!");
+        Console.WriteLine("This all you can eat experience only costs $9.99 per guest!");
         Console.WriteLine("Here at Greasy Jean's water is always free or upgrade to a 32oz softdrink for just $2.00!");
-        AddPadding(Padding);
+        AddPadding(padding);
 
         //Warn Guests About Group Size Limit//
         Console.ForegroundColor = ConsoleColor.Red;
         Console.WriteLine("                     ****Please note**** \n" +
             "Due to current staffing shortages we must limit party size to 6 guests or less");
-        AddPadding(Padding);
+        AddPadding(padding);
         Console.ForegroundColor = ConsoleColor.White;
 
         //Get Number Of Guests In The Group//
 
         Console.WriteLine("How many guests are in your party?");
-        GuestCount = int.Parse(Console.ReadLine());
+        var guestCount = int.Parse(Console.ReadLine());
```

Comment:
Some of the variables made sense being inline as they were declared to be closer to their logic.